src/dataflow/app-config.json

```diff
@@ -11,9 +11,6 @@
   "defaultDocumentTemplate": {
     "tiles": [
       {
-        "layout": {
-          "height": 500
-        },
```

Comment:
I forgot to ask earlier, but does this have any negative effect on sizing for new documents?

Reply:
This was removed so that we could meet the following story requirement: 
"Program tiles will take up the full vertical screen by default."
`getEditorStyle` in the program editor is now used to set the initial tile height.  However, some additional testing wouldn't hurt to see if there are adverse consequences.